dev/test-fpindex-api.py

```
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
+"""
+Test script for fpindex API
+Demonstrates the current API endpoints that our proxy service needs to mimic
+"""
+
+import requests
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)